### PR TITLE
LSIR: update

### PR DESCRIPTION
**Pull request by tharvik**
Following #136

### CODE REVIEW

File: data/LSIR/projects.yaml [hunk 1]
```diff
@@ -67,75 +77,85 @@ projects:
     information:
       - type: Demo
         title: Online Technology Demos
-        url: https://modemos.epfl.ch/
-      - type: Demo
-        title: Online Image Retrieval Demo
-        url: https://modemos.epfl.ch/article
```
Comment (cgrigis) [inline data/LSIR/projects.yaml:73]: If I'm not mistaken, this demo is not linked from the "main" page

File: data/LSIR/projects.yaml [hunk 2]
```diff
@@ -67,75 +77,85 @@ projects:
...
       - Artificial Intelligence
     type: Framework
     date_added: 2019-03-18
-    date_updated: 2021-01-22
+    date_updated: 2021-04-22
 
   ai-data-platform-v2:
     name: AI-Driven Classifier Building Pipeline
     description: Platform for speeding up the development of Artificial-Intelligence classifiers
-    url: https://www.epfl.ch/labs/lsir/ai-driven-classifier-building-pipeline/
```
Comment (cgrigis) [inline data/LSIR/projects.yaml:88]: That's a matter for a long debate, whether to end URLs with a slash or not  ;-)

**Review by cgrigis — COMMENTED** [review]
Minor comments, otherwise LGTM

File: data/LSIR/projects.yaml (hunk 1 above)
Comment (tharvik) [reply data/LSIR/projects.yaml:73]: You're right, it's not; I'm putting it back.

File: data/LSIR/projects.yaml (hunk 2 above)
Comment (tharvik) [reply data/LSIR/projects.yaml:88]: Héhé, indeed it is, as you saw, I prefer not having slash at the end, but it's clearly a matter of taste :)

**Review by cgrigis — APPROVED** [review]
LGTM

**Pull request merged**